### ISSUE TITLE
Quest: Cuergo's Gold

### ISSUE THREAD

**Issue by Kundaro** [elysium-project/server#1986]
The item called Cuergo's Treasure Map that begins this quest is not properly starting the quest. Can I get some help? Thank you!

**Comment by Phoosy** [comment]
Use the search function next time. 
Already reported here: https://github.com/elysium-project/server/issues/1677
Getting addressed here: https://github.com/elysium-project/server/pull/1979

**Comment by Kundaro** [comment]
@Phoosy I uhh, don't know what that means. Just want to be able to accept this damned quest.

**Comment by Phoosy** [comment]
Github is not the platform to seek assistance, it is for development to address bugs and push fixes to the server. Individualized help can be given using the in-game Help Menu (Red ? in your default UI bar). In this case, the quest issue was already reported and the fix is being evaluated to be pushed to the development branch and eventually live.

tl;dr  you're not getting personal help here.

**Comment by Kundaro** [comment]
@Phoosy Yeah about that, the GM I contacted in game told me to come here for a fix.

**Comment by Phoosy** [comment]
They would have indicated for you to _report_ the issue, but otherwise it will not be fixed until something is pushed to the servers by development. I would know, I was a GM that told players the exact same thing.

**Comment by thepparker** [comment]
Duplicate #1150
